<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Making 3D tiff file</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonDirectory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input file directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirectory">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>1061</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>1111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>661</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output 3D image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>1061</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCompress">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMake3D">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Make 3D tiff</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTemplate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_template">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Template of 2D tiff files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(e.g. frame%04d.tif )</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_firstnum">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lastnum">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>99</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_firstnum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First file number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lastnum">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Last file number</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxBits">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_8bits">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>8 bits</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_16bits">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>16 bits</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outputFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMake3D</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>make3Dtiffer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDirectory</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>directorySelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>outputFileSelecter()</slot>
  <slot>make3Dtiffer()</slot>
  <slot>directorySelecter()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroupMethod"/>
 </buttongroups>
</ui>
